DirectProgramming/DPC++FPGA/Tutorials/Features/prototype/CSR_and_streaming_interfaces/sample.json

```diff
@@ -0,0 +1,84 @@
+{
+  "guid": "69415BED-D452-449A-8F5A-DB01ACCE38DC",
```

Comment:
Just to make sure: did you generate a new guid?

Reply:
yup!